types/stream-to-blob/stream-to-blob-tests.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types = "node"/>
+
+import { Stream } from "stream";
+import streamToBlob from "stream-to-blob";
```

Comment:
I believe you should use the `import = ... ` syntax form here instead, to match the export syntax that was used:
`import streamToBlob = require("stream-to-blob");`

Reply:
@gabritto thank for review, I fixed it as per your request.